hd/README.md

```diff
@@ -0,0 +1,44 @@
+## Introduction
+
+This program adds `# ` to the beginning of the stdin and prints it.
+
+## Implementations
+
+- C (Does not support trim)
```

Comment:
Since I do not speak any other languagues except C, I'm going to review only this implementation.